src/Card/CardSection/index.js.flow

```diff
@@ -2,21 +2,24 @@
 import type { ReactComponentStyled } from "styled-components";
 
 import type { Globals } from "../../common/common.js.flow";
-import typeof CardSectionHeaderType from "./CardSectionHeader/index.js.flow";
-import typeof CardSectionContentType from "./CardSectionContent/index.js.flow";
 
 export type Props = {|
-  +children: React$Node,
+  +title?: React$Node,
+  +icon?: React$Node,
+  +description?: React$Node,
+  +dataA11ySection?: string,
```

Comment:
Is `dataA11ySection` really necessary? cc @filipdanisko (btw it's not in README.md)